nucypher/utilities/networking.py

```diff
@@ -200,4 +201,60 @@ def determine_external_ip_address(
     # complete failure!
     if not rest_host:
         raise UnknownIPAddress('External IP address detection failed')
+
     return rest_host
+
+
+def _is_valid_ipv4(ip: str) -> bool:
+    try:
+        ip = ip_address(ip.strip())
+        return isinstance(ip, IPv4Address) and ip.is_global
+    except AddressValueError:
+        return False
+
+
+def _ipv4_to_ipv6(ip: str) -> Optional[str]:
```

Comment:
shouldn't this be called `_ipv4_from_ipv6` or  `_ipv6_to_ipv4`?

Reply:
Yes! fixed.